packages/app/pages/index.tsx

```diff
@@ -84,6 +137,9 @@ const MembershipCard = (props: {
       });
     }
   }, [setPassport, passport, web3]);
+  useEffect(() => {
+    setUrl(props.redirect_url);
+  }, [props.redirect_url]);
```

Comment:
this looks unnecessary. Why not just use `redirect_url` directly?

Reply:
oh I see redirect_url is the initial value and it could change based on when it loads